### DISCUSSION TITLE
Fluxstore Delivery App

### DISCUSSION THREAD

**Discussion by rohangaurat** [inspireui/support#5132] [Working on Features]
Hi Team,
We are user of WCFM.
And they have feature - Delivery Person & App.
Buy we want you developed App as Fluxstore Delivery App.

I hope, you will focus on this development.

Thank you

Best Regards.
Rohan

**Comment by ekramul-1** [comment]
Please develop delivery from inspireui team. I already have but we need that from inspireui team.

**Comment by CeoAlexMartinez** [comment]
esperando esta caracteristica
